packages/dashboard/src/components/querySettingsSync/refreshRateDropDown.tsx

```diff
@@ -0,0 +1,44 @@
+import { FormField, Select } from '@cloudscape-design/components';
+import React, { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { onChangeRefreshRateAction } from '~/store/actions';
+import { DashboardState } from '~/store/state';
+import { selectIntialRefreshRate } from './utils';
+
+type SelectedOption = {
+  label?: string;
+  value?: string;
+};
+
+export const RefreshRateDropDown = () => {
+  const dispatch = useDispatch();
+  const refreshRate = useSelector(
+    (state: DashboardState) =>
+      state.dashboardConfiguration.querySettings.refreshRate
+  );
+  const [selectedRefreshRate, setSelectedRefreshRate] =
+    useState<SelectedOption>(selectIntialRefreshRate(refreshRate));
+
```

Comment:
Why do you need this state to store the option? It seems like additional state for the selected option could be avoided by creating the option each render.